src/PlatformInstaller/NativeMethods/ShowWindowCommands.cs

```diff
@@ -0,0 +1,83 @@
+﻿internal enum ShowWindowCommands : int
```

Comment:
I copied it from SC installer,  There was no link to the origin in that code base